engine/apps/schedules/models/on_call_schedule.py

```diff
@@ -207,7 +208,16 @@ class OnCallSchedule(PolymorphicModel):
     )
 
     name = models.CharField(max_length=200)
+
+    # TODO: drop this field in a subsequent release, this has been migrated to slack_channel field
     channel = models.CharField(max_length=100, null=True, default=None)
+    slack_channel = models.ForeignKey(
+        "slack.SlackChannel",
+        null=True,
+        default=None,
+        on_delete=models.SET_NULL,
+        related_name="+",
```

Comment:
what does `related_name="+"` do? is it some kind of alias for smth?

Reply:
![Screenshot 2024-11-04 at 12 13 30 PM](https://github.com/user-attachments/assets/8d3b89cb-6751-4dbc-be44-601660ed687f)

so basically for a `SlackChannel` instance, there won't be a `schedules` attribute to interact w/ all on-call schedules associated w/ that slack channel (probably not needed for now, but we can change in the future if need be).